x-pack/plugins/siem/public/endpoint_hosts/store/reducer.ts

```diff
@@ -10,26 +10,24 @@ import { AppAction } from '../../common/store/actions';
 import { ImmutableReducer } from '../../common/store';
 import { Immutable } from '../../../common/endpoint/types';
 
-export const initialHostListState = (): HostState => {
-  return {
-    hosts: [],
-    pageSize: 10,
-    pageIndex: 0,
-    total: 0,
-    loading: false,
-    error: undefined,
-    details: undefined,
-    detailsLoading: false,
-    detailsError: undefined,
-    policyResponse: undefined,
-    policyResponseLoading: false,
-    policyResponseError: undefined,
-    location: undefined,
-  };
+export const initialHostListState: Immutable<HostState> = {
```

Comment:
i changed this, but should I have?

Reply:
did you get any failures in TS checks 😆 ? 
I think that ok to make this change since that is the direction we're going. I think this was not immutable to begin with because we had started off without using Immutable and we had not had an opportunity to refactor it.

Reply:
ok sounds good.